datanode/datanode.go

```diff
@@ -396,6 +457,9 @@ func (d *dataNode) ID() string {
 }
 
 func (d *dataNode) Serve() {
+	// wait for server is ready to serve
```

Comment:
add back the timer to see how much time take to recover?

Reply:
it is already in the Serve function